packages/cmd/src/utils/CommandUtils.ts

```diff
@@ -74,28 +73,25 @@ export class CommandUtils {
         return command.trim();
     }
 
-
     /**
      * Check if an option was specified by the user.
      * @param optionName - the option to check for.
-     * @param {Arguments} commandArguments: The arguments specified by the user.
+     * @param {ICommandArguments["args"]} args: The arguments specified by the user.
      * @param {ICommandDefinition} commandDefinition - the definition for the command
      * @returns {boolean} true: the option was specified by the user.
      *                    false: the option was omitted/set to false
      */
-    public static optionWasSpecified(optionName: string, commandDefinition: ICommandDefinition,
-                                     commandArguments: Arguments): boolean {
+    public static optionWasSpecified(optionName: string, commandDefinition: ICommandDefinition, args: ICommandArguments["args"]): boolean {
         const optionDef = CommandUtils.getOptionDefinitionFromName(optionName, commandDefinition);
-
         if (isNullOrUndefined(optionDef)) {
             // if it's not an option, it's not specified
             return false;
         }
         if ((optionDef.type as string) === "boolean") {
-            return commandArguments[optionName] === true;
+            return args[optionName] !== undefined;
```

Comment:
Why the special treatment for the boolean types?
Is there a reason why you would want it to be `null`?
Why not just call the `isNullOrUndefined` function with `args[optionsName]`?